### PR TITLE
Switching to Nuxt

### PR DESCRIPTION
**Pull request by kezlya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
-# AntHive.IO
+# AntHive
 
-AntHIve.IO is a turn-base coding game. Command your ants to collect food and return it to your hive to grow your colony. A player with the biggest colony and less errors wins the game. For detailed rules please visit [Rules Section ](https://anthive.io/rules/)
+> Turn-based codding game
```
Comment (darthkimo) [inline README.md]: I think you mean Turn-based coding rignt?

File: pages/leaderboard.vue [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  background-size: contain;
+  background-repeat: no-repeat;
+}
+.profile-chip-version {
+  width: 48px;
+  height: 48px;
+  float: right;
+  border-radius: 100%;
+  color: rgba(255, 255, 255, 1);
+  background-color: rgba(95, 73, 47, 0.9);
```
Comment (darthkimo) [inline pages/leaderboard.vue]: Remove leading 0 and just use .9 instead of 0.9
background-color: rgba(95, 73, 47, .9);

File: README.md (hunk 1 above)
Comment (kezlya) [reply README.md]: Done. Thank you

File: pages/leaderboard.vue [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+  mounted() {
+    this.$http.get('https://search.anthive.io/users/_search').then(response => {
+      this.APIdata = response.data.hits.hits
+    })
+  }
+}
+</script>
+
+<style scoped>
+.profile-chip {
```
[the remark this answers is not in the document]
Comment (kezlya) [reply pages/leaderboard.vue]: Acknowledge. Would you recommend an automated tool for that?

File: pages/leaderboard.vue (hunk 2 above)
Comment (kezlya) [reply pages/leaderboard.vue]: @boris-veshkin please address in the next PR

**Pull request merged**